code/_compile_options.dm

```diff
@@ -70,4 +70,8 @@
 #endif
 
  #define SERVICE_CMD_PARAM_KEY "serviceCommsKey"
- 
\ No newline at end of file
+
+//Update this whenever the db schema changes 

```

Comment:
But ya, this file is live tracked, so i wouldn't need to update it manually or anything.